packages/flutter_test/lib/src/matchers.dart

```diff
@@ -354,6 +354,50 @@ Matcher coversSameAreaAs(Path expectedPath, { required Rect areaToCompare, int s
 /// ```
 /// {@end-tool}
 ///
+/// By default, the Flutter framework uses a font called 'Ahem' which shows
+/// squares instead of characters. To make golden tests more valuable, there
+/// is the possibility to render images using other more human-readable fonts.
+/// For example, this is how to load the 'Roboto' font for a golden test:
```

Comment:
Please include a note that fonts may render slightly differently on different platforms and between versions of Flutter, which may cause unintended image diffs.

Reply:
Sure thank you for the feedback